ocs/defaults.py

```diff
@@ -21,69 +20,51 @@
 ROOK_CLUSTER_NAMESPACE = 'openshift-storage'
 KUBECONFIG_LOCATION = 'auth/kubeconfig'  # relative from cluster_dir
 API_VERSION = "v1"
+CEPHFILESYSTEM_NAME = 'ocsci-cephfs'
 
 TEMP_YAML = os.path.join(constants.TEMPLATE_DIR, "temp.yaml")
 
-TOOL_POD_DICT = load_yaml_to_dict(
-    os.path.join(
-        constants.TEMPLATE_DEPLOYMENT_DIR, "toolbox_pod.yaml"
-    )
+TOOL_POD_DICT = os.path.join(
+    constants.TEMPLATE_DEPLOYMENT_DIR, "toolbox_pod.yaml"
 )
-CEPHFILESYSTEM_DICT = load_yaml_to_dict(
-    os.path.join(
-        constants.TEMPLATE_CSI_FS_DIR, "CephFileSystem.yaml"
-    )
-)
-CEPHBLOCKPOOL_DICT = load_yaml_to_dict(
-    os.path.join(
-        constants.TEMPLATE_DEPLOYMENT_DIR, "cephblockpool.yaml"
-    )
+
+CEPHFILESYSTEM_YAML = os.path.join(
+    constants.TEMPLATE_CSI_FS_DIR, "CephFileSystem.yaml"
 )
-CSI_RBD_STORAGECLASS_DICT = load_yaml_to_dict(
-    os.path.join(
-        constants.TEMPLATE_CSI_RBD_DIR, "storageclass.yaml"
-    )
+
+CEPHBLOCKPOOL_YAML = os.path.join(
+    constants.TEMPLATE_DEPLOYMENT_DIR, "cephblockpool.yaml"
 )
-CSI_CEPHFS_STORAGECLASS_DICT = load_yaml_to_dict(
-    os.path.join(
-        constants.TEMPLATE_CSI_FS_DIR, "storageclass.yaml"
-    )
+
+CSI_RBD_STORAGECLASS_DICT = os.path.join(
+    constants.TEMPLATE_CSI_RBD_DIR, "storageclass.yaml"
 )
-CSI_PVC_DICT = load_yaml_to_dict(
-    os.path.join(
-        constants.TEMPLATE_PV_PVC_DIR, "PersistentVolumeClaim.yaml"
-    )
+
+CSI_CEPHFS_STORAGECLASS_DICT = os.path.join(
```

Comment:
All these variables are now just paths to a file. I wonder if we should rename them appropriately? Maybe ``CSI_CEPHFS_STORAGECLASS_PATH``?

Reply:
Done